lib/active_merchant/billing/gateways/first_pay/first_pay_json.rb

```diff
@@ -75,6 +81,7 @@ def scrub(transcript)
           gsub(%r(("processorId\\?"\s*:\s*\\?")[^"]*)i, '\1[FILTERED]').
           gsub(%r(("merchantKey\\?"\s*:\s*\\?")[^"]*)i, '\1[FILTERED]').
           gsub(%r(("cardNumber\\?"\s*:\s*\\?")[^"]*)i, '\1[FILTERED]').
+          gsub(%r(("paymentCryptogram\\?"\s*:\s*\\?")[^"]*)i, '\1[FILTERED]').
```

Comment:
Can we include a test to validate that we are scrubbing this value correctly?

Reply:
Done!